Add readmes to notebooks (#184)
openvinotoolkit/openvino_notebooks: CONTRIBUTING.md

- `200-` OpenVINO model demos: demonstrate inference on a particular model.
- `300-` Training notebooks: notebooks that include code to train neural networks.

### Writing Readme

Every notebook must have readme file briefly describing content of related tutorial. To keep it simple structure is described below:

``` markdown

> # Title of tutorial

Reviewer: ```suggestion
# Title of Tutorial
```

[brief intro, basic information about what will be described]

> ## Content table

Reviewer: ```suggestion
## Notebook Contents
```

> [more details, possibly information about papers or used models, data]

Reviewer: ```suggestion
[more details, possibly information about research papers, the model(s) used and/or data]
```

> Here will be placed other subsections, e.g license information.

Reviewer: ```suggestion
Additional subsections, e.g license information.
```

## Installation Instructions
[link to installation guide, other important information for install process]
```

### File structure

> To keep form similar for each and every notebook please use form as described

Reviewer: ```suggestion
To maintain consistency between notebooks, please follow the directory structure outlined below.
```

```markdown
<three-digit-number>-<title>/
├── README.md
├── <three-digit-number>-<title>.ipynb
├── utils/
├── model/
└── data/
```

> In case of output provided by Notebook please create folder ```local``` on the same level as readme file.

Reviewer: ```suggestion
If the Jupyter notebook generates output data, like images or video, please a create folder named ```output``` at the same level as the README.md file.
```

## Validation

> ### Automated tests

Reviewer: ```suggestion
### Automated Tests
```
